src/components/modal/AposModal.vue

```diff
@@ -91,7 +91,6 @@ export default {
       if (!this.hasLeftRail && this.hasRightRail) {
         return 'apos-modal__main--grid apos-modal__main--with-right-rail';
       }
-      return 'apos-modal__main--no-rails';
```

Comment:
Now I'm wondering if we should return _something_, like `null` or `false`.

Reply:
returning false here, produces no output